### PR TITLE
Igorvereshch

### PR DESCRIPTION
**Pull request by kolyasalubov**

### CODE REVIEW

File: HW3/igorvereshch/homework3_task2_no_lists.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Вирішив спробувати розв'язати це завдання (лекція 3, завдання 2) без списку
+# бо маю багато вільного часу і не знав про Codewars.
+# Ще є такий момент що цей випадок спрацює і з числами меншими від 1000
+# але тоді всі відсутні цифри будуть присвоєні 0
+# Якщо знайдете число яке не спрацює, пишіть.
```
Comment (kolyasalubov) [inline HW3/igorvereshch/homework3_task2_no_lists.py:5]: alright)

File: HW4/igorvereshch/homework4_task2.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Не знаю чи ця конструкція переганяє наш матеріал
+# Але в розв'язках в Codewars вона зустрічається постійно
+list1 = [i ** 2 - 3 * i - 6 for i in range(10)]
```
Comment (kolyasalubov) [inline HW4/igorvereshch/homework4_task2.py:3]: list comprehension

Comment (igorvereshch) [reply HW4/igorvereshch/homework4_task2.py:3]: Thank you.

File: HW3/igorvereshch/homework3_task2_no_lists.py (hunk 1 above)
Comment (igorvereshch) [reply HW3/igorvereshch/homework3_task2_no_lists.py:5]: Glad you liked it

**Pull request merged**